LICENSE.md

```diff
@@ -28,16 +28,14 @@ Under the following terms:
   so in any reasonable manner, but not in any way that suggests the
   licensor endorses you or your use.
 
-**No additional restrictions**---You may not apply legal terms or
+* **No additional restrictions**---You may not apply legal terms or
 technological measures that legally restrict others from doing
 anything the license permits.  With the understanding that:
 
-Notices:
-
-* You do not have to comply with the license for elements of the
+  * You do not have to comply with the license for elements of the
   material in the public domain or where your use is permitted by an
   applicable exception or limitation.
-* No warranties are given. The license may not give you all of the
+  * No warranties are given. The license may not give you all of the
```

Comment:
These two list items appeared to be rendered correctly before. Or were they broken @Fienne ?

![image](https://user-images.githubusercontent.com/304786/196552203-32f34bf8-b068-4e39-8b45-54a10dd10a5d.png)

https://github.com/common-workflow-language/user_guide/blob/f9e63daf90cb84fb23e5df678592528b451263e0/LICENSE.md

Reply:
They were rendered correctly. I thought they are sub lists to the No additional restrictions list and had to be arranged in a way that is more readable like so: 
![Screenshot from 2022-10-18 23-00-27 1](https://user-images.githubusercontent.com/72187339/196554025-cd1d823c-034a-4116-9887-d2a7c999212c.png)

Reply:
Ah! Now I see. They were not related to the paragraph above with "Notices:"… I think. It was really confusing, great catch. I think your version is correct, but leaving it up to @tetron or @mr-c or @swzCuroverse to confirm. Thanks for clarifying!

Reply:
Thank you :)